vita3k/app/src/app_init.cpp

```diff
@@ -75,6 +75,22 @@ void update_viewport(EmuEnvState &state) {
     if (h > 0) {
         const float window_aspect = static_cast<float>(w) / h;
         const float vita_aspect = static_cast<float>(DEFAULT_RES_WIDTH) / DEFAULT_RES_HEIGHT;
+#ifdef __APPLE__
+        int windowWidth, windowHeight;
+        SDL_GetWindowSize(state.window.get(), &windowWidth, &windowHeight);
+        float scale = (float)w / windowWidth;
```

Comment:
I think using snake_case would be more consistent here.
And while it's not strictly enforced everywhere, the Vita3K also prefers snake_case.